policykit/policyengine/models.py

```diff
@@ -760,10 +760,6 @@ def save(self, *args, **kwargs):
 
 class PlatformActionBundle(BaseAction):
 
-    bundled_actions = models.ManyToManyField(PlatformAction)
```

Comment:
wait why are we removing this?

Reply:
In the original, there were two copies of this line so I removed the first. The same text is present on line 771.